themes/default/content/blog/organization-access-tokens/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Launching Organization Access Tokens for the Pulumi Service"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-04-28T9:00:00-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: As the adoption of the Pulumi Service grows, with 350% growth in the number of enterprise customers using it in the last year, the size of customers using the Pulumi Service has also grown. To empower customers using the Pulumi Service at scale, today we are launching Organization Access Tokens. 
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - devon-grove
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+    - pulumi-enterprise
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+As the adoption of the Pulumi Service grows, with 350% growth in the number of enterprise customers using it in the last year, the size of customers using the Pulumi Service has also grown. To empower customers using the Pulumi Service at scale, today we are launching Organization Access Tokens. 
```

Comment:
I like this statement, but something about the first sentence is not rolling off the tongue for me. Here's an alternative to consider:

> As enterprise adoption of the Pulumi Service has grown 350% over the last year, we've seen a strong customer demand for tools to manage automated Pulumi use cases such as CI/CD and Automation API at scale. Today we are launching Organization Access Tokens to empower our largest customers to manage automated workloads in a secure and collaborative manner.